components/var_dumper/advanced.rst

```diff
@@ -50,28 +50,33 @@ corresponding Data object could represent only a subset of the cloned variable.
 Before calling :method:`Symfony\\Component\\VarDumper\\Cloner\\VarCloner::cloneVar`,
 you can configure these limits:
 
-* :method:`Symfony\\Component\\VarDumper\\Cloner\\VarCloner::setMaxItems`
-  configures the maximum number of items that will be cloned
-  *past the first nesting level*. Items are counted using a breadth-first
-  algorithm so that lower level items have higher priority than deeply nested
-  items;
-* :method:`Symfony\\Component\\VarDumper\\Cloner\\VarCloner::setMaxString`
-  configures the maximum number of characters that will be cloned before
-  cutting overlong strings;
-* in both cases, specifying `-1` removes any limit.
+:method:`Symfony\\Component\\VarDumper\\Cloner\\VarCloner::setMaxItems`
+    configures the maximum number of items that will be cloned
+    *past the first nesting level*. Items are counted using a breadth-first
+    algorithm so that lower level items have higher priority than deeply nested
+    items;
+
+:method:`Symfony\\Component\\VarDumper\\Cloner\\VarCloner::setMaxString`
+    configures the maximum number of characters that will be cloned before
+    cutting overlong strings;
+
+In both cases, specifying `-1` removes any limit.
```

Comment:
you have to use a double backtick in reStructured Text to create an inline code block: ``` ``-1`` ```